src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  height: 408px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
Consider using a variable for the border color '#f3f3f3' to maintain consistency and ease of maintenance if the color scheme changes in the future.